bokehjs/src/coffee/tool/inspectors/inspect_tool.coffee

```diff
@@ -28,7 +28,8 @@ class InspectTool extends Tool.Model
   event_type: "move"
 
   nonserializable_attribute_names: () ->
-    super().concat(['event_type', 'inner_only'])
+    attrs = _.without(super(), 'active')
+    return attrs.concat(['event_type', 'inner_only'])
```

Comment:
So, this is why I limited my self to only inspect tools, because that's the simple case. With other kinds of tools, it's more complex and would require, I presume, cumbersome logic. A tool manager is a solution, but staying with `active` property for all tools would work too, just with cumbersome logic. My first approximation to encoding active toolset is the following descriptor `Tuple(GestureTool, Set(InspectTool))`.